Vietnamese translation 1st PR

Anh-La opened the pull request:
add Vietnamese Translation
update Readme file per @kdmukai

Anh-La committed:
1st Vi translation commit

add Vietnamese Translation
update Readme file per @kdmukai

Anh-La committed:
2nd Vi commit

Fix the job failed due to a KeyError: 'độ dài' in the test tests/screenshot_generator/generator.py::test_generate_all[vi]. This error indicates that the string 'độ dài' is missing from the relevant translation dictionary or data structure used in the test for the Vietnamese locale.

Anh-La committed:
3rd Vi commit

Shorten phrases to fit small screen

Anh-La committed:
4th Vi commit

Shorten and expand for artistic view and meaning

kdmukai commented:
Please split this into two PRs: One for the README and one for the Vietnamese translations.

The README PR will need a little more explanation; at first I didn't even remember that these instructions weren't in this repo yet.

kdmukai commented:
Note: These translations were NOT in transifex. I was able to upload them from this PR into Transifex.

Please review the README steps and download the messages.po from transifex and replace the file in this PR with the transifex version. They should be more or less identical, but I just want to guarantee that everything remains fully consistent and in sync with transifex.

bitcoinprecept commented:
I'm glad to see you incorporated the proposed fixes to Keith's Readme file. Like Keith said, if you can split the documentation update our to it's own PR, we can get that addressed on it's own.

Now that Keith has backfilled your translations into Transifex, ideally we can identify someone else to serve in the Reviewer role to check what you have proposed there (both the strings, and ideally the screenshots).

Anh-La committed:
Revert README file to its original state

Anh-La commented:
The README.md file has been reverted to its original state.

This branch now exclusively contains the newly added Vietnamese translation. 

***The messages.po file used for this translation was downloaded directly from Transifex, as advised.***

Anh-La committed:
update the reviewed message

Anh-La committed:
Replace Capital letters in buttons

Replace capital letters from message.po file downloaded from transifex with lower case in buttons.
Already informed admin

kdmukai commented:
Vietnamese is currently at 83% translated on Transifex after the most recent source string update in Dec 2025.

Anh-La commented:
the updated version is now 100% translated on Transifex in 24 Feb 2026.

Next step, assign to a reviewer to finalise.

kdmukai commented:
Known minor overflow issue below.

I opened a PR to @Anh-La to add the latest changes from Transifex: https://github.com/Anh-La/seedsigner-translations/pull/1

That will bring this PR up to match Transifex's 96% translated state.

ACK for v0.8.7, even if my PR isn't merged into this PR (if we merge first, I'll just edit/re-create my PR to target the main repo).

---

<img width="528" height="388" alt="DonateView_overflow" src="https://github.com/user-attachments/assets/a283e7a9-b0c3-4208-a3ea-c70fcb839598" />

newtonick committed:
Merge branch 'dev' into Vietnamese

newtonick commented:
ACK
